src/ol/graticule.js

```diff
@@ -442,11 +442,11 @@ ol.Graticule.prototype.handlePostCompose_ = function(e) {
   var i, l, line;
   for (i = 0, l = this.meridians_.length; i < l; ++i) {
     line = this.meridians_[i];
-    vectorContext.drawLineStringGeometry(line, null);
+    vectorContext.drawLineString(line, null);
```

Comment:
We can still use the unexported methods internally.  This is akin to us exporting `observable.on()` and using `ol.events.listen()` internally.  This gives us stricter type checking internally.